R/dvhydrograph-render.R

```diff
@@ -287,6 +300,11 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", minMaxE
                            lty=ifelse(plotItem[['newSet']] == "est", 1, 6), col=ifelse(plotItem[['newSet']] == "est", "orange", "red3")),
                       styles$est_lines)
     ),
+    estimated4Edges = list(
+      arrows = append(list(x0=plotItem[['time']], x1=plotItem[['time']], y0=plotItem[['y0']], y1=plotItem[['y1']], 
+                           lty=ifelse(plotItem[['newSet']] == "est", 1, 7), col=ifelse(plotItem[['newSet']] == "est", "black", "red3")),
```

Comment:
Yeah in this spot we pulled the color out of styles because it is dynamic (creating the vertical lines connecting estimated/non-estimated series and choosing whether or not the line should be the estimated color or non-estimated color). If there is a clean way to move the color back into the styles that would be cool, but I don't think we came up with anything at the time this was originally added.

Do you have any ideas @lindsaycarr ?